config/default/kustomization.yaml

```diff
@@ -0,0 +1,74 @@
+# Adds namespace to all resources.
+namespace: volume-replication-operator-system
+
+# Value of this field is prepended to the
+# names of all resources, e.g. a deployment named
+# "wordpress" becomes "alices-wordpress".
+# Note that it should also match with the prefix (text before '-') of the namespace
+# field above.
+namePrefix: volume-replication-operator-
```

Comment:
We need to be careful about names ending up being too long. Can we cut it short?

Reply:
Created https://github.com/kube-storage/volume-replication-operator/issues/7